_articles/es-ES/best-practices.md

```diff
@@ -0,0 +1,299 @@
+---
+locale: es-ES
+title: Buenas Prácticas para Mantenedoras
+description: Pensadas para facilitarte la vida como mantenedora de código abierto, desde el proceso de documentación hasta cómo construir una comunidad provechosa.
+class: best-practices
+toc:
+  what-does-it-mean-to-be-a-maintainer: "¿Qué significa ser mantenedora?"
+  documenting-your-processes: "Documentar tus procesos"
+  learning-to-say-no: "Aprender a decir no"
+  leverage-your-community: "Construir una comunidad provechosa"
```

Comment:
"provechosa" me chirria un poco pero no sabría como traducir "leverage" sin parecer que usamos la comunidad para nuestro beneficio :thinking:

Reply:
Pues compartitmos la misma duda. Elegí "provechosa" como la opción que me pareció menos mala.

Se me ocurre que la expresión "sacar partido" es una manera suave de decirlo, que puede encajar. "Cómo construir una comunidad a la que sacarle partido".

Otra opción: "Cómo construir una comunidad útil".

La última que se me ocurre "Cómo construir una comunidad productiva"

Reply:
Fua me lo has puesto peor ahora. Démosle vueltas y igual podemos usarlas en distintos contextos.

Reply:
Propongo "Construye el apoyo de tu comunidad". y rehago la traducción en el campo semántico del apoyo. En lugar de "to leverage your contributors": "para apoyarte en tus contribuidores" o "para conseguir apoyo de tus contribuidores". 

¿Qué te parece?